bundles/commerce-mock-0.1.0/chart/commerce-mock/templates/mf-commerce.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: "ui.kyma-project.io/v1alpha1"
+kind: MicroFrontend
+metadata:
+  name: commercermicrofrontend
+  namespace: <namespace>
+  labels:
+    app: commercemicrofrontend
+spec:
+  appName: commerce-microfrontend
+  displayName: Commerce View Configuration
+  version: v1
+  category: <category>

```

Comment:
What is the effect of that field and what can be a cluster independent value?

Reply:
changed it to Mocks